Makefile

```diff
@@ -9,7 +9,7 @@
 GO=CGO_ENABLED=0 GO111MODULE=on go
 
 test:
-	$(GO) test ./... -cover
+	$(GO) test ./... -coverprofile=coverage.out
```

Comment:
Can you add coverage.out to the .gitignore please?

Reply:
Good catch, updated PR with that change